neo4j/tests/common.py

```diff
@@ -1,9 +1,10 @@
-import os
+# import os
```

Comment:
If this and the other line aren't needed, can they be removed instead of commented?